pom.xml

```diff
@@ -110,6 +110,11 @@
             <optional>true</optional>
         </dependency>
         <dependency>
+            <groupId>com.jcabi</groupId>
+            <artifactId>jcabi-aspects</artifactId>
+            <optional>true</optional>
```

Comment:
no, we shouldn't use it in this project. any extra dependency should be avoided in this project